server/testdata/tools/xlam.gotmpl

```diff
@@ -0,0 +1,47 @@
+{{ if .System }}{{ .System }}
+{{ end }}
+{{- range $i, $m := .Messages }}
+{{- if eq .Role "user" }}
+{{- $last := (or (eq (len (slice $.Messages $i)) 1) (eq (len (slice $.Messages $i)) 2)) -}}
```

Comment:
This line trims right so if you end up removing it, you'll need to join `### Instruction` with the previous `{{- if eq .Role "user" }}`